Privacy Policy.md

```diff
@@ -0,0 +1,60 @@
+# WHATWG Website Privacy Policy
+
+Last updated April 20, 2020
+
+WHATWG respects your privacy and is committed to protecting your personal data. This privacy policy covers how we collect, use, and disclose your personal information when you visit our website, https://whatwg.org.
```

Comment:
It seems this is intended to cover other subdomains of whatwg.org as well?

Reply:
It is intended to cover subdomains.

Reply:
I've suggested a "and subdomains" addition in https://github.com/whatwg/sg/pull/120/files#r417184376